ooiservices/app/logs/ooiservices.log

```diff
@@ -940,3 +940,44 @@
+2015-01-12 21:46:36,465 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:36,772 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:37,172 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:37,509 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:37,921 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:38,217 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:38,617 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:39,111 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:39,461 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:39,977 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:40,380 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:40,832 - 19537 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:53,215 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:53,964 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:54,017 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:54,519 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:54,864 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:55,300 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:55,686 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:56,090 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:56,501 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:56,929 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:57,327 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:57,667 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:58,156 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:58,719 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:59,072 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:59,518 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:46:59,997 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:47:00,390 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:47:00,853 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:47:01,311 - 19599 - app - __init__:42 - INFO - Application Process Started
+2015-01-12 21:47:01,700 - 19599 - app - __init__:42 - INFO - Application Process Started
```

Comment:
logs need to be added to the `.gitignore` file so we don't include them in our commits please.